recipe/conda_build_config.yaml

```
@@ -0,0 +1,4 @@
+occt:
+  - 7.7.0
+  - 7.7.1=*novtk*
+  - 7.7.1=*all*
```

```suggestion
  - 7.7.1 *novtk*
  - 7.7.1 *all*
```

The equal sign in the filename is breaking Windows because the variable expands to an assignment, I think. CMD is fun 😂 

Good news is that you don't need equal signs here; a space is enough.

Commit this and rerender!